rocAL/rocAL_pybind/amd/rocal/plugin/pytorch.py

```diff
@@ -276,44 +204,45 @@ class ROCALClassificationIterator(ROCALGenericIterator):
     fill_last_batch = True, last_batch_padded = True   -> last batch = [7, 7], next iteration will return [1, 2]
     fill_last_batch = True, last_batch_padded = False  -> last batch = [7, 1], next iteration will return [2, 3]
     """
+
```

Comment:
it is better not to combine code reformatting with other changes relevant to this PR

Reply:
reverted the code reformating in pytorch.py